src/fragments/dropdowns.mdx

```diff
@@ -0,0 +1,15 @@
+The Dropdown component provides state and accessibility through the context API to its consumers while enabling
+customization.
+
+**Placement / Positioning**: Internally, the Dropdown component uses [PopperJS](https://popper.js.org/docs/v2/)
+for positioning. Use the `popperModifiers` prop of the Menu component to customize the placement, boundaries,
+overflows etc.
+
+**Dropdown State**: The [Downshift](https://github.com/downshift-js/downshift) `render-prop` library handles the
+keyboard and accessibility logic. Provide other customizations directly to the Downshift provider via the
+`downshiftProps` prop. See the [stateReducer](https://github.com/downshift-js/downshift#statereducer) pattern
+for an example of a common customization strategy.
+
+**Server Side Rendering**: Server side rendering might require configuring specific Downshift settings. The
+package mentioned re-exports the Downshift `resetIdCounter` utility, resetting the internal id counter which
+is used to generate unique ids for Downshift.
```

Comment:
Could we include `resetIdCounter` as part of the possible imports here:
<img width="717" alt="Screen Shot 2021-09-29 at 11 45 46 AM" src="https://user-images.githubusercontent.com/1975639/135330219-049304e7-7427-473d-8b6f-29dc7e2a6e19.png">

Reply:
It appears that this list is not meant to be an exhaustive list of all possible imports from the package, only an example? For instance, ItemMeta, MediaItem, MediaFigure, MediaBody etc are all valid subcomponents to use within an Autocomplete Menu, but aren't listed here. Could someone correct me if I'm wrong?

Reply:
If that's the case, then we can forget about this comment. Unless @jzempel has other ideas we can let this point go.

Reply:
This list is intentionally populated via the frontmatter for the page, which should consist of the components most endemic to the main component of interest. Go to https://garden.zendesk.com/components/menu#configuration for the fuller listing more directly related to `Menu`.

This config is currently displaying as desired.